crates/fluvio-extension-common/src/output/table.rs

```diff
@@ -57,20 +56,16 @@ where
 
         // Create the table
         let mut table = Table::new();
-        let mut format = *format::consts::FORMAT_CLEAN;
-        let pad_left = if indent { 5 } else { 1 };
-        format.padding(pad_left, 1);
-        table.set_format(format);
 
         // add header
-        table.set_titles(header);
+        table.set_header(header);
 
         // add rows
         for row in content {
             table.add_row(row);
         }
 
         // print table to stdout
-        table.printstd();
+        println!("{table}");
```

Comment:
Instead of hard coding to `println`, should add similar API to comfy_table

Reply:
Do you mean that I should implement a function on `Table` that does that?

Reply:
Easiest way is to add custom trait

Reply:
update?